packages/backend-modules/auth/lib/AccessToken.js

```diff
@@ -139,7 +139,7 @@ const resolve = async (token, { pgdb }) => {
   if (userId && scope && expiresAt && hmac) {
     const payload = getPayload({ userId, scope, expiresAt })
     const userRecord = await pgdb.public.users.findOne({ id: userId })
-    if (userRecord.revokedAt) {
+    if (userRecord.deletedAt) {
```

Comment:
This will throw an error if `findOne` returns `null`.

```
TypeError: Cannot read properties of null (reading 'deletedAt')
```

Reply:
Albeit code might work eventually, maybe revert to previous code and extend query like so:

```
   const key = await pgdb.public.users.findOneFieldOnly(
      { id: userId, 'deletedAt !=': null },
      'accessKey',
    )
```